source/blog/2016-07-29-last-week-in-manageiq-saying-goodbye-to-rubyrep.html.md

```diff
@@ -0,0 +1,122 @@
+
+We found an issue with replication when rows which belonged to a region were migrated.
+The join table received an id in the global region when it should have had a remote region
+id assigned.
+
+`cloud_subnets_network_ports` prior to the change:
+
+`vmdb_production=# select * from cloud_subnets_network_ports;
+       id       | cloud_subnet_id | network_port_id | address 
+----------------+-----------------+-----------------+---------
+ 99000000000001 |               2 |               1 | 
+ 99000000000002 |               3 |               2 | 
+ 99000000000003 |               3 |               3 | 
+ 99000000000004 |               3 |               4 | 
+ 99000000000005 |               3 |               5 |` 
+
+The same table after the change:
+
+`vmdb_production=# select * from cloud_subnets_network_ports;
+ id | cloud_subnet_id | network_port_id |    address     
+----+-----------------+-----------------+----------------
+  1 |               2 |               1 | 192.0.2.1
+  2 |               3 |               2 | 192.0.2.2
+  3 |               3 |               3 | 192.0.2.3
+  4 |               3 |               4 | 192.0.2.4
+  5 |               3 |               5 | 192.0.2.5`
+
+## New
+
+### [Expose generic object and definition to automate](https://github.com/ManageIQ/manageiq/pull/9861)
+
+New automate models for `GenericObjectDefinition` and `GenericObject` have been added.
+
+### [Create a bundle service template and attach to blueprint](https://github.com/ManageIQ/manageiq/pull/9854)
+
+Part of the Blueprint feature.  This PR builds a new method `create_bundle` in the `Blueprint` class.
+It creates a bundle service template and places all copied item templates underneath it. All of these items are part
+of a single Blueprint.
+
+## Deleted
+
+### [These files should not be executable](https://github.com/ManageIQ/manageiq/pull/10068)
+
+Less about 'deletions' and more about security. This fix turned off the executable `0755` permission for 78 files!
+
+### [MiqPolicyController - remove dead policy_options and cat_pressed](https://github.com/ManageIQ/manageiq/pull/10112)
+
+The method `policy_options` was only used on vms and not for policies as the name implied. It also called the 
+
+previously removed `build_policy_tree`.  Due to the above the method was removed.
+
+`cat_pressed` is no longer used anywhere in the codebase and was [happily](https://github.com/himdel) removed!
+
+## Wrapping up
+
+Of the [100][PRs merged last week] PR's merged last week, I obviously only presented a small sampling. Look out for the next installment of *Last Week in ManageIQ!*
+
+[PRs merged last week]: https://github.com/ManageIQ/manageiq/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222016-07-25+..+2016-07-29%22+sort%3Acreated-desc&utf8=%E2%9C%93
+[Commits merged last week]: https://github.com/manageiq/manageiq/compare/master@%7B2016-07-25%7D...@%7B2016-07-29%7D
```

Comment:
Guess we could probably nix this line if it's not used. I guess since GitHub won't support showing as many commits as we put out there in a week, this link might be of limited value anyway for any upcoming issues?